.github/workflows/helm-release.yml

```diff
@@ -0,0 +1,41 @@
+name: ci-latest-helm-chart-release
+
+on:
+  push:
+    branches: [master]
+    paths:
+      - "helm/**"
+
+jobs:
+  publish-chart:
+    name: Update Stable Helm Chart With Latest Changes
+    if: ${{ (github.repository == 'kubearmor/sidekick') && (!contains(github.event.head_commit.message, '[skip ci]')) }}
+    runs-on: ubuntu-20.04
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: azure/setup-helm@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Update Chart.yaml
+        run: |
+          STABLE_VERSION=`cat STABLE-RELEASE`
+          sed -i -e "s/appVersion:.*/appVersion: $STABLE_VERSION/g" helm/sidekick/Chart.yaml
+          sed -i -e "s/version:.*/version: $STABLE_VERSION/g" helm/sidekick/Chart.yaml
```

Comment:
Will we be releasing a stable version of sidekick? If yes, then let's also add the STABLE_VERSION file.